CONTRIBUTING.md

```diff
@@ -1,39 +1,65 @@
-# Contributing
+# How to Contribute
 
-## How to Contribute
+1. Report bugs as github issues. We don't use a template, but try to provide
+   some sensible information such as what what happened, what you expected to
```

Comment:
> what what

Reply:
Well... :) we can have a template if you want

Reply:
No i mean its literally in the changed text: "as what what happened" :D